dashboards/google-kubernetes-engine/gke-workloads-at-risk.json

```diff
@@ -0,0 +1,365 @@
+                "visible": true
+              }
+            ],
+            "dataSets": [
+              {
+                "timeSeriesQuery": {
+                  "timeSeriesQueryLanguage": "fetch k8s_container\n| metric 'kubernetes.io/container/memory/request_bytes'\n| filter resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp)-system'\n| group_by 5m, [value_request_bytes_mean: mean(value.request_bytes)]\n| every 5m\n| group_by\n    [resource.project_id, resource.location, resource.cluster_name,\n     resource.namespace_name,\n     metadata.system.top_level_controller_name:\n       metadata.system_labels.top_level_controller_name,\n     metadata.system.top_level_controller_type:\n       metadata.system_labels.top_level_controller_type],\n    [value_request_bytes_mean_mean: mean(value_request_bytes_mean)]\n| filter val() == 0"
+                }
+              }
+            ],
+            "metricVisualization": "NUMBER"
+          },
+          "title": "Memory (reliability risk) - workloads without memory requested"
+        },
+        "width": 6,
+        "xPos": 0,
+        "yPos": 4
+      },
+      {
+        "height": 4,
+        "widget": {
+          "timeSeriesTable": {
+            "columnSettings": [
+              {
+                "column": "resource.project_id",
+                "visible": true
+              },
+              {
+                "column": "resource.location",
+                "visible": true
+              },
+              {
+                "column": "resource.cluster_name",
+                "visible": true
+              },
+              {
+                "column": "resource.namespace_name",
+                "visible": true
+              },
+              {
+                "column": "metadata.system.top_level_controller_name",
+                "visible": true
+              },
+              {
+                "column": "metadata.system.top_level_controller_type",
+                "visible": true
+              }
+            ],
+            "dataSets": [
+              {
+                "timeSeriesQuery": {
+                  "timeSeriesQueryLanguage": "fetch k8s_container\n| metric 'kubernetes.io/container/cpu/request_cores'\n| filter resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp)-system'\n| group_by 5m, [value_request_cores_mean: mean(value.request_cores)]\n| every 5m\n| group_by\n    [resource.project_id, resource.location, resource.cluster_name,\n     resource.namespace_name,\n     metadata.system.top_level_controller_name:\n       metadata.system_labels.top_level_controller_name,\n     metadata.system.top_level_controller_type:\n       metadata.system_labels.top_level_controller_type],\n    [value_request_bytes_mean_mean: mean(value_request_cores_mean)]\n| filter val() == 0"
+                }
+              }
+            ],
+            "metricVisualization": "NUMBER"
+          },
+          "title": "CPU (reliability risk) - workloads without CPU requested"
```

Comment:
Shorten this to "Workloads without CPU requested"

Reply:
updated